### PR TITLE
feat(grid block): adding a grid component

### LINKED ISSUE #133: Need Gradient BG for Account Managent services image
I used an image that has a transparent BG, Need a gradient background or maybe just a drop shadow.....see screenshot
Screenshot 2023-08-15 at 5 44 21 PM
 our site: https://main--takeda-ihs--hlxsites.hlx.page/service/
Live site: https://takvirtualexperience.com/service/

### PR DESCRIPTION
**Pull request by lwintch**
adding support for a grid on the services page with a background gradient section and including the clip-path on the hero when the section has an inverted gradient background.

Fix #133

Test URLs:

- Before: https://main--takeda-ihs--hlxsites.hlx.page/service?lighthouse=on
- After: https://fix-133-gradient-bg--takeda-ihs--hlxsites.hlx.page/drafts/lwintch/service?lighthouse=on

### CODE REVIEW

File: scripts/third-party.js [hunk 1]
```diff
@@ -13,5 +13,5 @@ function createInlineScript(innerHTML, parent) {
 
 // eslint-disable-next-line import/prefer-default-export
 export function integrateMartech() {
-  createInlineScript(GTM_SCRIPT, document.body);
+  (window.location.href.match(/^https?:\/\/(localhost|127.0.0.1)/) === null) && createInlineScript(GTM_SCRIPT, document.body);
```
Comment (lwintch) [inline scripts/third-party.js:16]: QoL to keep the console a little cleaner when developing locally.

File: styles/styles.css [hunk 2]
```diff
@@ -444,6 +444,21 @@ main .section.gray-gradient-background[data-background-image] {
   );
 }
 
+main .angled-inverted-background.section.hero-container > .hero-wrapper {
+  background-color: var(--gray-gradient-light);
+
+}
+
+main .section.angled-inverted-background.hero-container .hero {
+  clip-path: polygon(0 0, 100% 0, 100% calc(100% - 5vw), 0 100%);
```
Comment (lwintch) [inline styles/styles.css:453]: `clip-path` and `background` values copied from live site.

File: styles/styles.css [hunk 3]
```diff
@@ -526,6 +541,10 @@ main .section.gray-gradient-background[data-background-image] {
   :root {
     --nav-height: 75px;
   }
+
+  main .section.angled-inverted-background.hero-container .hero {
+    clip-path: polygon(0 0, 100% 0, 100% calc(100% - 10vw), 0 100%);
```
Comment (lwintch) [inline styles/styles.css:546]: Breakpoints translated from `10vw` at `< 768px` and `5vw` `>= 768px` to be `10vw` below `900px` breakpoint and `5vw` at `900px` breakpoint and above.

**Comment by bstopp** [comment]
I thought the intent/request was just to support the gradient background for the image, no need for the new block?

File: scripts/third-party.js (hunk 1 above)
Comment (lwintch) [reply scripts/third-party.js]: I implemented this change.

**Comment by lwintch** [comment]
> #133

I believe our latest site just had that section of services as an image.  As I was working on this page, I thought I would make this a component we could use instead of just an image.  If that's wrong, I can remove it no problem.

**Review by bstopp — APPROVED** [review]

**Pull request merged**